src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -222,6 +222,8 @@ public void periodic() {
     Logger.getInstance().recordOutput("Drive/getHeading", getYaw);
     Logger.getInstance().recordOutput("Drive/getPitch", pitch);
 
+    //System.out.println("Position:   " + getX() + " " + getY());
```

Comment:
[nit] I would recommend adding debug mode for printing out the traces. It can be resolved later